downpour_downloader/books.py

```diff
@@ -101,7 +101,9 @@ def _create_book(self, book):
 
         # Setup dates
         purchase_date = parser.parse(attrs['data-purchase-date'])
-        release_date = parser.parse(attrs['data-release-date']).date()
+        release_date = parser.parse(attrs['data-release-date']).date() if attrs['data-release-date'] else None
```

Comment:
Sadly, data-release-data now is populated lazily when clicking on a library entry, so we can't rely on its presence.